packages/react-server/package.json

```diff
@@ -24,50 +24,50 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "cls-q": "1.1.0",
-    "cookie": "0.1.2",
+    "cookie": "^0.3.1",
     "cookie-parser": "1.4.3",
-    "eventemitter3": "0.1.6",
-    "express-state": "1.2.0",
+    "eventemitter3": "^2.0.2",
+    "express-state": "^1.4.0",
     "lodash": "^4.16.4",
-    "mobile-detect": "1.3.0",
+    "mobile-detect": "^1.3.5",
     "q": "1.4.1",
     "querystring": "0.2.0",
-    "request-local-storage": "1.1.0",
-    "routr": "0.1.3",
-    "winston": "2.2.0"
+    "request-local-storage": "^1.1.1",
+    "routr": "^2.1.0",
```

Comment:
The tests didn't catch this because all of our tests only send GET requests, no POST/HEAD/etc.  I added some unit tests, but we should probably add full integration tests for at least POST requests.